src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer.UnitTests/Utilities/AbstractLanguageServerHostTests.cs

```diff
@@ -61,7 +59,7 @@ internal static async Task<TestLspServer> CreateAsync(ClientCapabilities clientC
         internal LanguageServerHost LanguageServerHost { get; }
         public ExportProvider ExportProvider { get; }
 
-        internal ServerCapabilities ServerCapabilities => _serverCapabilities ?? throw new InvalidOperationException("Initialize has not been called");
+        internal ServerCapabilities ServerCapabilities { get => field ?? throw new InvalidOperationException("Initialize has not been called"); private set; }
```

Comment:
I like this so much. That it allows `field` to be nullable before it is initialized. I am thinking of all the places I have nullable fields that are only nullable because they aren't set in the constructor.